src/storage/mongodb/BillingStorage.ts

```diff
@@ -234,4 +237,61 @@ export default class BillingStorage {
     // Debug
     await Logging.traceEnd(tenantID, MODULE_NAME, 'deleteInvoice', uniqueTimerID, { id });
   }
+
+  // ------------------------------------------------------------------------------------------
+  // BILLING SETTINGS
+  // ------------------------------------------------------------------------------------------
+  public static async getBillingSettings(tenantID: string): Promise<BillingSettings[]> {
+    const settings = await SettingStorage.getSettings(tenantID, { identifier: TenantComponents.BILLING }, Constants.DB_PARAMS_SINGLE_RECORD);
+    if (settings && settings.count > 0) {
+      const allBillingSettings: BillingSettings[] = settings.result.map((setting) => BillingStorage.convertToBillingSettings(setting));
+      return allBillingSettings;
+    }
+    return null;
+  }
+
+  public static async getBillingSetting(tenantID: string, settingID: string): Promise<BillingSettings> {
+    const settings = await SettingStorage.getSettings(tenantID, { settingID }, Constants.DB_PARAMS_SINGLE_RECORD);
+    if (settings?.result?.[0]?.content) {
```

Comment:
Use the DB_PARAMS_DEFAULT_RECORD and ensure that you have only one record here like in other Storage class.